app/code/Magento/Backend/Ui/Component/Control/SaveSplitButton.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Backend\Ui\Component\Control;
+
+use Magento\Framework\View\Element\UiComponent\Control\ButtonProviderInterface;
+use Magento\Ui\Component\Control\Container;
+
+/**
+ * @api
+ */
+class SaveSplitButton implements ButtonProviderInterface
+{
+    /**
+     * @var string
+     */
+    private $saveTarget;
+
+    /**
+     * @param string $saveTarget
+     */
+    public function __construct($saveTarget)
+    {
+        $this->saveTarget = $saveTarget;
```

Comment:
`saveTarget` is pretty uncommon, I found only `\Magento\ConfigurableProduct\Block\Adminhtml\Product\Edit\Button\Save::getSaveTarget` with grep. Can it be `targetName`?

Reply:
Fixed